alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/processor/service/ProcessorService.java

```diff
@@ -1,5 +1,9 @@
 package eu.xenit.alfresco.processor.service;
 
 public interface ProcessorService {
+    enum Scope {
```

Comment:
what is `Scope` ?

Reply:
`TransactionScope` ? as in: transactions to scan :thinking:

Reply:
but we always scan ALL transactions, from start till the end - there is no *new* concept ?

Reply:
Removed!